### PR TITLE
CircleCI parallelisation

### PR DESCRIPTION
**Pull request by damacus**
# Description

Speed up CircleCI runs. Currently CircleCI takes 45minutes to run the full test suite. 

## Fixes 

- #350 

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by damacus** [comment]
To note, I'm seeing lots of errors in systemd when poise runs `systemctl daemon-reload`

[See here for an example that occurs pretty regularly](https://circleci.com/gh/sous-chefs/haproxy/1575?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)

**Comment by JohnRoesler** [comment]
And they fail inconsistently. Can you re run them?

**Comment by damacus** [comment]
I'm just going to merge this in, and depoise this.

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a <a href="https://github.com/sous-chefs/line-cookbook/blob/master/CHANGELOG.md">CHANGELOG</a> entry.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: .kitchen.yml [hunk 1]
```diff
@@ -41,6 +41,7 @@ suites:
         source_version: '1.6.14'
         source_url: 'http://www.haproxy.org/download/1.6/src/haproxy-1.6.14.tar.gz'  # yamllint disable-line
         source_checksum: 'bac949838a3a497221d1a9e937d60cba32156783a216146a524ce40675b6b828'  # yamllint disable-line
+    includes: [centos-6, centos-7, debian-8, ubuntu-16.04]
```
Comment (JohnRoesler) [inline .kitchen.yml]: Do you want Ubuntu 1804 here?

Comment (damacus) [reply .kitchen.yml]: That fails in the same way, becaue of a new version of openssl

**Pull request merged**